packages/core/src/Cardano/types/Utxo.ts

```diff
@@ -51,3 +51,10 @@ export interface TxOut {
 }
 
 export type Utxo = [HydratedTxIn, TxOut];
+
+/**
+ * Gets whether the given datum is a datum hash.
+ *
+ * @param datum The datum to be checked for.
+ */
+export const isDatumHash = (datum: unknown): datum is DatumHash => datum !== null && typeof datum === 'string';
```

Comment:
I think a more appropriate place for this is under Serialization, because core types have a separate property for datum and datum hash. Or maybe in tx-builder package, unless you can think of other non-tx builder use cases for this util

Reply:
It seems we already have it in Serialization, just not exported, done here: https://github.com/input-output-hk/cardano-js-sdk/pull/1299/commits/cb58b451fe765d28473496f13cfcdbeaad9b3fb3